Epam.TestAutomation/Epam.TestAutomation.BDD/Steps/SearchSteps.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Threading;
+using Epam.TestAutomation.BDD.DataDrivenTestingTests;
+using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Utilities.Logger;
+using Epam.TestAutomation.Helper;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow;
+using Epam.TestAutomation.BDD.Steps.GeneralSteps;
+using Epam.TestAutomation.BDD.Steps.Pages;
+using Epam.TestAutomation.BDD.Steps.SharedComponents;
+using Epam.TestAutomation.BDD.Steps;
+using Epam.TestAutomation.BDD;
+using Epam.TestAutomation.Utilities;
+using Epam.TestAutomation.Web.PageObjects.Pages;
+
+namespace Epam.TestAutomation.BDD
+{
+    [Binding]
+    public class SearchByKeywordSteps
+    {
+        private string enteredKeyword;
+        private string selectedLocationCountry;
+        private string selectedLocationCity;
+        private string selectedSkillsOne;
+        private string selectedSkillsTwo;
+        private MainPage LandingPage => new();
+         
+        [Given(@"I navigate to the main page of the Epam website")]
+        public void GivenINavigateToLandingPageOfEpamSite()
+        {
+            Logger.Info("Step: Navigate to Landing Page of Epam site");
+            BrowserFactory.Browser.GotToUrl(TestSettings.ApplicationUrl);
+        }
+
+        [When(@"I accept all cookies on Epam website")]
+        public void WhenIAcceptAllCookiesOnEpamSite()
+        {
+            Logger.Info("Step: Accept all cookies on Epam Site");
+            Thread.Sleep(3000);
+            LandingPage.AcceptAllCookies();
+        }
+
+        [When(@"Go to the “Careers” “Join our Team” section")]
+        public void GivenIAcceptAllCookiesOnEpamSite()
+        {
+            Logger.Info("Step: Go to the “Careers” -> “Join our Team” section");
+            BrowserFactory.Browser.Action.MoveToElement(BrowserFactory.Browser.FindElement(Locators.CareersButton)).Build().Perform();
+            BrowserFactory.Browser.FindElement(Locators.JobListingsLink).Click();
```

Comment:
it would be better to create methods  landingPage..OpenCarrersMenu() and landingPage.ClickJobListingButton()